src/FieldWarning/Assets/Model/Game/MatchSession.cs

```diff
@@ -131,6 +147,18 @@ public void Awake()
 #endif
         }
 
+        public void Start()
+        {
+            
+            if (_loadedData == null)
+            {
+                LoadingScreen.destinationScene = SceneManager.GetActiveScene().buildIndex;
+                SceneManager.LoadScene(4, LoadSceneMode.Single);
```

Comment:
loading a scene by index like this is error-prone since we will change the scene ordering and count around, you can also load by name